scipy/optimize/_numdiff.py

```diff
@@ -240,17 +252,24 @@ def approx_derivative(fun, x0, method='3-point', rel_step=None, f0=None,
 
         Note, that sparse differencing makes sense only for large Jacobian
         matrices where each row contains few non-zero elements.
+    return_linear_operator : boolean, optional
+        When `return_linear_operator` is True returns an LinearOperator.
```

Comment:
I think slight clarification is necessary what this linear operator will do. On the one hand it is obvious (if you think about it), on the other hand it can be somewhat confusing when you see it for the first time.